rust/agents/relayer/src/msg/metadata_builder.rs

```diff
@@ -140,10 +142,17 @@ impl MetadataBuilder for LegacyMultisigIsmMetadataBuilder {
             )
             .await.context(CTX)?
         else {
-            info!(
-                ?validators, threshold, highest_known_nonce,
-                "Could not fetch metadata: Unable to reach quorum"
-            );
+            if validators.is_empty() {
+                info!(
+                    ism=%multisig_ism.address(),
+                    "Could not fetch metadata: Unable to reach quorum because no validators are defined"
```

Comment:
Rather than defined, perhaps "no validator set for chain X is configured on the recipients ISM"